docs/reference/aggregations/metrics/weighted-avg-aggregation.asciidoc

```diff
@@ -12,6 +12,16 @@ As a formula, a weighted average is the `∑(value * weight) / ∑(weight)`
 
 A regular average can be thought of as a weighted average where every value has an implicit weight of `1`.
 
+[NOTE]
+======
+While multiple values-per-field are allowed, only one weight is allowed.  If the aggregation encounters
+a document that has more than one weight (e.g. the weight field is a multi-valued field) it will throw an exception.
+If you have this situation, you will need to specify a `script` for the weight field, and use the script
+to combine the multiple values into a single value to be used.
+
+This single weight will be applied independently to each value extracted from the `value` field.
```

Comment:
I wonder if its worth having an example of a single weight being applied to each value independantly to help solidify what we mean?